Core/src/org/sleuthkit/autopsy/datasourcesummary/ui/DataSourceSummaryTabbedPane.java

```diff
@@ -45,7 +45,7 @@
     private final List<Pair<String, BaseDataSourceSummaryPanel>> tabs = Arrays.asList(
             Pair.of(Bundle.DataSourceSummaryTabbedPane_detailsTab_title(), new DataSourceSummaryDetailsPanel()),
             Pair.of(Bundle.DataSourceSummaryTabbedPane_countsTab_title(), new DataSourceSummaryCountsPanel()),
-            Pair.of(Bundle.DataSourceSummaryTabbedPane_detailsTab_title(), new DataSourceSummaryUserActivityPanel())
+            Pair.of(Bundle.DataSourceSummaryTabbedPane_userActivityTab_title(), new DataSourceSummaryUserActivityPanel())
```

Comment:
I have already fixed this in my PR, might want to remove this so there is no conflicts.

Reply:
I'm hoping it shouldn't be an issue since hopefully it should be the exact same change.  It seems like this change would be a good change to make for finishing up the User Activity tab.

Reply:
I wasn't suggesting reverting it, just change it back, but what ever you want to do.